### ISSUE TITLE
Stress test the gateway

### ISSUE THREAD

**Issue by justinmoon** [fedimint/fedimint#1930]
Blocked on https://github.com/fedimint/fedimint/issues/1447 since our current gateway isn't designed to operate concurrently.

Once we have a gateway running the new client library, let's write some tests to really hammer the gateway and understand what kind of throughput it can achieve and what breaks first with load.

**Comment by m1sterc001guy** [comment]
Gateway is parallel now. @douglaz has written a load test tool that runs in CI and makes LN payments. Should we keep this open? Maybe we could someone generate a daily or weekly report based on the output of our average payment/median payment time (or other stats)?

**Comment by justinmoon** [comment]
Might be nice to have a test that attempts like 100 concurrent payments and just makes sure they all succeed in reasonable time?

**Comment by justinmoon** [comment]
I think we can close this based on @douglaz load tests
